app/src/main/java/net/gsantner/opoc/frontend/base/GsFragmentBase.java

```diff
@@ -70,12 +69,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         view.postDelayed(() -> {
-            synchronized (_fragmentFirstTimeVisibleSync) {
-                if (getUserVisibleHint() && isVisible() && _fragmentFirstTimeVisible) {
-                    _fragmentFirstTimeVisible = false;
-                    onFragmentFirstTimeVisible();
-                }
-            }
+            checkRunFirstTimeVisible();
             attachToolbarClickListenersToFragment();
         }, 1);
```

Comment:
Should probably only call from onResume - that is when the fragment is fully ready and displayed